package.json

```diff
@@ -16,6 +16,7 @@
     "test:nsp": "nsp check",
     "test:a11y": "echo 'a11y tests not implemented'",
     "sonar-scanner": "node_modules/sonar-scanner/bin/sonar-scanner",
+    "sonar-scan": "node_modules/sonar-scanner/bin/sonar-scanner",
```

Comment:
Both line 18 and 19 essentially do the same thing. Can you please remove one of them.